docs/notebooks/frequency_analysis.ipynb

```diff
@@ -8,7 +8,7 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        pr, t=20, dist="genextreme", mode="max", freq="Y", month=[5, 6, 7, 8, 9, 10]
</pre><p>Is there are reason to use <code>freq='Y'</code>  and not <code>YS</code> ?  It seems a bit odd as <code>YS</code> is almost universally used elsewhere  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/xclim/pull/1338/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>